core/tracker/tracker_internal_test.go

```diff
@@ -1402,6 +1402,6 @@ func TestSubmittedProposals(t *testing.T) {
 				Version: eth2spec.DataVersionDeneb,
 			},
 		}, 1*time.Millisecond)
-		require.ErrorContains(t, err, "could not determine if proposal was synthetic or not")
+		require.ErrorContains(t, err, "determine if proposal was synthetic")
```

Comment:
To be approved only if the prod code change is approved.
```suggestion
		require.ErrorContains(t, err, "could not determine if proposal was synthetic or not")
```